apps/src/weblab/CdoBramble.js

```diff
@@ -312,7 +332,7 @@ export default class CdoBramble {
   }
 
   createProjectRootDir(callback) {
-    this.fileSystem().mkdir(this.projectPath, err => {
+    this.shell().mkdirp(this.projectPath, err => {
```

Comment:
note about this change: `this.shell().mkdirp` will recursively create directories -- `this.fileSystem().mkdir` does not. this is necessary for when we reformat the filesystem (or, possibly, on the first ever project creation, but i haven't been able to repro that).

`this.projectPath` is `/codedotorg/weblab/:channelId/`, so after reformatting the filesystem, `this.fileSystem().mkdir` would fail because `codedotorg/` and its subdirectory `weblab/` don't exist. `this.shell().mkdirp` will create all 3 directories.